s3_scrubber/src/main.rs

```diff
@@ -111,13 +69,25 @@ async fn tidy(
     depth: TraversingDepth,
     skip_validation: bool,
 ) -> anyhow::Result<()> {
-    let binary_name = env::args()
-        .next()
-        .context("binary name in not the first argument")?;
-
     let dry_run = !cli.delete;
-    let _guard = init_logging(&binary_name, dry_run, node_kind.as_str());
-    let _main_span = info_span!("tidy", binary = %binary_name, %dry_run).entered();
+    let file_name = if dry_run {
+        format!(
+            "{}_{}_{}__dry.log",
+            CLI_NAME,
+            node_kind,
+            chrono::Utc::now().format("%Y_%m_%d__%H_%M_%S")
+        )
+    } else {
+        format!(
+            "{}_{}_{}.log",
+            CLI_NAME,
+            node_kind,
+            chrono::Utc::now().format("%Y_%m_%d__%H_%M_%S")
+        )
+    };
+
+    let _guard = init_logging(&file_name);
+    let _main_span = info_span!("tidy", %dry_run).entered();
```

Comment:
> Warning: In asynchronous code that uses async/await syntax, Span::enter may produce incorrect traces if the returned drop guard is held across an await point. See the method documentation for details. 

https://docs.rs/tracing/latest/tracing/span/index.html

=> Make this an `#[instrument(...)]` on top of `async fn tidy`. Since `dry_run` is defined inside the function body, use `field::Empty` it as exampled in https://docs.rs/tracing/latest/tracing/index.html#using-the-macros

Reply:
Removed this span.